source/help/getting-started/organizing-your-sidebar.rst

```diff
@@ -0,0 +1,183 @@
+
+Organizing Your Sidebar
+=======================
+
+Conversations in Mattermost are crucial to company productivity and success. Keeping conversations organized in the sidebar creates an efficient workplace. 
+
+Channel types
+-------------
+
+Public channels
+~~~~~~~~~~~~~~~
+
+Public channels are open to everyone on a team. New team members are automatically added to two Public channels when they sign up: ``Town Square`` and ``Off-Topic``.
+
+Private channels
+~~~~~~~~~~~~~~~~
+
+Private channels are for sensitive topics and are only visible to select team members. In Team Edition, any member of a Private channel can add or remove other members from Private channels, but in Enterprise Edition `these permissions can be restricted to the Channel Admins and System Admins <https://docs.mattermost.com/help/getting-started/managing-members.html#user-roles>`__.
+
+Direct Messages and Group Messages
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use a Direct Message when you want a private conversation with one person. Use a Group Message when you want to set up a private conversation with up to seven other participants. If you want to communicate with more than seven people, you must create a Private channel. Direct and Group Messages are visible only to the people involved.
+
+If your System Admin has allowed it, you can start a Direct Message or a Group Message with people on other teams.
+
+Customizing your sidebar
+------------------------
+
+You can organize your channel sidebar based on how you use Mattermost. Customizations you make are only visible to you and won't affect what your teammates see in their sidebars.
+
+Here’s how your sidebar is set up by default:
+
+- All Public and Private Channels you've joined are listed in the **Channels** category, sorted alphabetically.
+- All your Direct Messages and Group Messages are listed in the **Direct Messages** category, sorted by recent activity.
+
+What can you customize?
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can customize your sidebar in the following ways:
+
+- Group and order channels into collapsible categories you create.
+- Sort channels in each category manually, alphabetically, or by recent activity.
+- Mute and unmute entire categories.
+- Filter your sidebar to view unread channels only, or choose to group unread messages into an **Unreads** category.
+- Sort Direct Messages alphabetically or by recent activity, and choose how many are displayed in your sidebar.
+
+  .. image:: ../../images/channel_sidebar_updates.gif
+
+Creating custom categories
+--------------------------
+
+Create custom categories to group channels together for quicker and easier navigation. For example, you can create a category called "Design" or "Marketing".
+
+Categories can be created in two ways:
```

Comment:
I propose that the two options be split into 1. and 2. One sentence for both options is a bit confusing and makes the next sentence seem like it's option two.

Reply:
I agree that updating the preceding text before describing the two ways of doing things is the best option, given the convention. I've updated both instances like this.